root/static/scripts/edit/externalLinks.js

```diff
@@ -439,6 +439,7 @@ export class ExternalLink extends React.Component<LinkProps> {
               />}
           </td>
         </tr>
+        {isEmpty(props) ||
```

Comment:
The indenting needs fixing here :) Also, we generally do `isEmpty(props) ? null : component` - both ways should work, but it seems more consistent to stick with that.

Reply:
In fact I was referring to the original code on line 431 for consistency, it is not my usual way, too. If we agree that this could be better we could change both lines. :)

Reply:
Oh, I see. Unless @yvanzo or @mwiencek see a reason to have it like this, I'd do the usual. But wait until they confirm :)

Reply:
It used to be the usual. @mwiencek can better explain why it changed. That is related to Flow IIRC.